### PR TITLE
Add new DefinitelyTyped JSPrintManager v2.0.2

### PR DESCRIPTION
**Pull request by auriou**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by auriou** [comment]
added https://neodynamic.com/products/printing/js-print-manager/ to print live from js

- Send Raw Bytes, Text & Files to any Client Printer
-  Print ESC/P, PCL, PostScript, ZPL, EPL, PDF, XLS, DOC
-  Thermal, impact, dot-matrix, RFID, kiosk, POS, receipt
-  Cross-browser & Plugin-free (Pure JavaScript)
-  Cross-Platform! Windows, Linux, RPi & Mac!

File: types/jsprintmanager/tslint.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "extends": "dtslint/dt.json" ,
+    "rules": {
```
Comment (armanio123) [inline types/jsprintmanager/tslint.json]: Is there any specific need for turning off this rules?

**Review by armanio123 — CHANGES_REQUESTED** [review]
Could you please add a test file?

**Comment by auriou** [comment]
I do not understand the error here, I added a test

File: types/jsprintmanager/tslint.json (hunk 1 above)
Comment (auriou) [reply types/jsprintmanager/tslint.json]: I removed the rules, and just kept the one for the interfaces

**Review by armanio123 — APPROVED** [review]

**Pull request merged**